src/forms/progressive-profiling-popup/index.jsx

```diff
@@ -93,7 +94,13 @@ const ProgressiveProfilingForm = () => {
   };
 
   const handleSelect = (e) => {
-    const { name, value } = e.target;
+    const { name, value, text } = e.target;
+
+    if (text === '') {
```

Comment:
is there any case where empty value is being selected?

Reply:
There are two things for the field, one is value and second is text. The value will be null or empty if the user selected text doesn't match with this countryList dict. And If user removes the text from the field, the text will be null and we are show error in case of no text in the field.